docs/configuration.rst

```diff
@@ -68,6 +68,7 @@ a given section and key.
    ==============  ======  =================================================================
    max_files       INT     maximum number of open files per client process
    flatten_writes  BOOL    enable flattening writes (optimization for overwrite-heavy codes)
+   local_extents   BOOL    enable extent tracking to service reads of local data
```

Comment:
Oh, well this was the table I was thinking of in the above comment.  Anyway, I think it would be nice to have a place to write more text for these options if needed, and then we could hyperlink to that from this table.

Reply:
Updated to ```service reads from local data if possible (default: off)```